### PR TITLE
V5.30 documentation

### PR DESCRIPTION
**Pull request by amyblais**
Needs one approval to be able to merge.

### CODE REVIEW

**Comment by amyblais** [comment]
@justinegeffen Can you help give this an approval so that I can merge it when needed 🙏 

(You can also add review comments after approving)

**Comment by justinegeffen** [comment]
Yep, on it! :)

**Review by justinegeffen — APPROVED** [review]

File: source/administration/changelog.md [hunk 1]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
 
 Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
+## Release v5.30
+
+### Compatibility
+  - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
```
Comment (justinegeffen) [inline source/administration/changelog.md:10]:
```suggestion
  - PostgreSQL ended long-term support for [version 9.4 in February 2020](https://www.postgresql.org/support/versioning). From v5.26 Mattermost officially supports PostgreSQL version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16, 2020). Please follow the instructions under the Upgrading Section within [the PostgreSQL documentation](https://www.postgresql.org/support/versioning/).
```

File: source/administration/changelog.md [hunk 2]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+#### Configure New Admin Roles Permissions in the System Console (E20, Beta)
+ - Mattermost [recently released three new pre-built granular administrator roles](https://mattermost.com/blog/mattermost-release-v5-28/) to enable you to selectively delegate administrative tasks to other members of your organization. The three new roles are System Manager, User Manager, and Read-only Admin. Now you can configure specific permissions for these roles directly from the System Console.
+
+### Improvements
+
+#### User Interface
+ - @-autocomplete results are now prioritized based on recency and thread activity.
+ - File attachments below the size of 10 (KB, MB, GB, TB, etc.) now allow showing fractions.
+ - The formatting of the channel header change message was improved.
+ - Team invite workflow now shows BOT tags when the search returns Bot users.
```
Comment (justinegeffen) [inline source/administration/changelog.md:26]:
```suggestion
 - Team invite workflow now shows BOT tags when the search returns bot users.
```

File: source/administration/changelog.md [hunk 3]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Added the ability to zoom in and out of PDF files.
+ - Added support for 16x16 base64 encoded mini images to use with progressive rendering.
+
+#### Notifications
+ - Channel-wide mentions are now automatically disabled when a user mutes a channel.
+
+#### Command Line Interface (CLI)
+ - Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks.
```
Comment (justinegeffen) [inline source/administration/changelog.md:34]:
```suggestion
 - Added new local API endpoints for getting, updating, and deleting incoming and outgoing webhooks.
```

File: source/administration/changelog.md [hunk 4]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Added ``mmctl system version`` endpoint to print the remote server version.
+ - Improved the ``mmctl system status`` command output to include all the reported values.
+
+#### Integrations
+ - Updated icon_emoji field in incoming webhooks to allow emojis to be specified with surrounding colons.
```
Comment (justinegeffen) [inline source/administration/changelog.md:39]:
```suggestion
 - Updated ``icon_emoji`` field in incoming webhooks to allow emojis to be specified with surrounding colons.
```

File: source/administration/changelog.md [hunk 5]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Dynamic auto-completion is now supported for built-in slash commands.
+ - Added plugin hooks for ``ReactionHasBeenAdded`` and ``ReactionHasBeenRemoved``.
+
+#### Administration
+ - Added the ability to load a set of custom configuration defaults from a MM_CUSTOM_DEFAULTS_PATH environment variable.
```
Comment (justinegeffen) [inline source/administration/changelog.md:44]:
```suggestion
 - Added the ability to load a set of custom configuration defaults from a ``MM_CUSTOM_DEFAULTS_PATH`` environment variable.
```

File: source/administration/changelog.md [hunk 6]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+#### Enterprise Edition (EE)
+ - Added the ability to retrieve compliance files from the System Console.
+
+### Bug Fixes
+ - Fixed an issue with broken link previews for Twitter links.
+ - Fixed an issue where editing a post did not submit with CMD+ENTER.
+ - Fixed an issue where users were able to create or edit slash commands to contain more than two slashes in the URL.
+ - Fixed an issue where resized emojis were being overwritten with original data.
+ - Fixed an issue where the sidebar category "More" menu was not shown when hovering over a long category name.
```
Comment (justinegeffen) [inline source/administration/changelog.md:55]:
```suggestion
 - Fixed an issue where the sidebar category **More** menu was not shown when hovering over a long category name.
```

File: source/administration/changelog.md [hunk 7]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where a received direct message did not show up on the sidebar if the direct message channel was newly created.
```
Comment (justinegeffen) [inline source/administration/changelog.md:56]:
```suggestion
 - Fixed an issue where a received Direct Message did not show up on the sidebar if the Direct Message channel was newly created.
```

File: source/administration/changelog.md [hunk 8]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where a search using from: failed to auto-load more results on the right-hand side when Elasticsearch was enabled.
```
Comment (justinegeffen) [inline source/administration/changelog.md:57]:
```suggestion
 - Fixed an issue where a search using **from:** failed to auto-load more results on the right-hand side when Elasticsearch was enabled.
```

File: source/administration/changelog.md [hunk 9]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where an s3 file backend TestFileConnection failed due to permissions if S3PathPrefix was in use.
```
Comment (justinegeffen) [inline source/administration/changelog.md:58]:
```suggestion
 - Fixed an issue where an s3 file backend ``TestFileConnection`` failed due to permissions if ``S3PathPrefix`` was in use.
```

File: source/administration/changelog.md [hunk 10]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Fixed an issue where an id was missing for a Tooltip in PostInfo.
```
Comment (justinegeffen) [inline source/administration/changelog.md:59]:
```suggestion
 - Fixed an issue where an ID was missing for a Tooltip in ``PostInfo``.
```

File: source/administration/changelog.md [hunk 11]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+
+### config.json
+Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in `config.json`, or the System Console when available.
```
Comment (justinegeffen) [inline source/administration/changelog.md:62]:
```suggestion
Multiple setting options were added to ``config.json``. Below is a list of the additions and their default values on install. The settings can be modified in ``config.json``, or the System Console when available.
```

File: source/administration/changelog.md [hunk 12]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+### Open Source Components
+ - Added ``@stripe/react-stripe-js``, ``@stripe/stripe-js``, and ``@types/country-list`` to https://github.com/mattermost/mattermost-webapp.
+ - Removed ``react-native-image-gallery`` from https://github.com/mattermost/mattermost-mobile.
+ - Added ``react-native-redash`` and ``react-native-share`` to https://github.com/mattermost/mattermost-mobile.
+ 
+### Websocket Event Changes
+ - In ``post_deleted`` websocket event, System Admins are now notified when a user initiates a post deletion.
+
+### API Changes
+ - Added new local API endpoints for getting, updating and deleting incoming and outgoing webhooks.
```
Comment (justinegeffen) [inline source/administration/changelog.md:82]:
```suggestion
 - Added new local API endpoints for getting, updating, and deleting incoming and outgoing webhooks.
```

File: source/administration/changelog.md [hunk 13]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Added new API endpoints to work with experimental collapsed threads.
+ 
+### Known Issues
+ - **Discard Changes** confirmation is not displayed when an admin adds people in the **System Roles** System Console page and clicks elsewhere before saving the changes.
+ - Error text is missing when the team name is left blank on team creation page.
```
Comment (justinegeffen) [inline source/administration/changelog.md:87]:
```suggestion
 - Error text is missing when the team name is left blank on the team creation page.
```

File: source/administration/changelog.md [hunk 14]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Line numbers do not line up with the text on code file previews.
+ - In some cases reply posts cannot be marked as unread.
+ - The threshold from the bottom of the screen is sometimes not respected for received messages.
+ - Posts created by bots containing attachments sometimes appear as repeated until the user refreshes the page.
+ - Emoji counter in the center channel doesn't always update immediately when a reaction is added in the right-hand side.
+ - A JavaScript error may appear in some cases when dismissing the new messages toast while scrolled up in the right-hand side.
+ - Admin Filter option is not disabled in AD/LDAP page for admin roles with ``sysconsole_write_authentication`` permission.
```
Comment (justinegeffen) [inline source/administration/changelog.md:94]:
```suggestion
 - The Admin Filter option is not disabled in AD/LDAP page for admin roles with ``sysconsole_write_authentication`` permission.
```

File: source/administration/changelog.md [hunk 15]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - On a server using a subpath, the URL opens a blank page if the System Admin changes the Site URL in the System Console. To fix this, the System Admin should restart the server.
+ - Login does not work when Custom Terms of Service is enabled and MFA is enforced.
+ - Google login fails on the Classic mobile apps.
+ - Status may sometimes get stuck as Away or Offline in High Availability mode with IP Hash turned off.
```
Comment (justinegeffen) [inline source/administration/changelog.md:98]:
```suggestion
 - Status may sometimes get stuck as **Away** or **Offline** in High Availability mode with IP Hash turned off.
```

File: source/administration/changelog.md [hunk 16]
```diff
@@ -4,6 +4,107 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
...
+ - Searching stop words in quotes with Elasticsearch enabled returns more than just the searched terms.
+ - Searching with Elasticsearch enabled may not always highlight the searched terms.
+ - Team sidebar on desktop app does not update when channels have been read on mobile.
```
Comment (justinegeffen) [inline source/administration/changelog.md:101]:
```suggestion
 - The team sidebar on the desktop app does not update when channels have been read on mobile.
```

File: source/administration/important-upgrade-notes.rst [hunk 17]
```diff
@@ -2,7 +2,7 @@ Important Upgrade Notes
...
+   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. Mattermost is officially supporting PostgreSQL version 10 with v5.26 release as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```
Comment (justinegeffen) [inline source/administration/important-upgrade-notes.rst:5]:
```suggestion
   PostgreSQL ended long-term support for `version 9.4 in February 2020 <https://www.postgresql.org/support/versioning>`_. From v5.26 Mattermost officially supports version 10 as PostgreSQL 9.4 is no longer supported. New installs will require PostgreSQL 10+. Previous Mattermost versions, including our current ESR, will continue to be compatible with PostgreSQL 9.4. PostgreSQL 9.4 and all 9.x versions are now fully deprecated in our v5.30 release (December 16). Please follow the instructions under the Upgrading Section within `the PostgreSQL documentation <https://www.postgresql.org/support/versioning/>`_.
```

File: source/administration/mmctl-cli-tool.rst [hunk 18]
```diff
@@ -4004,6 +4088,321 @@ mmctl version
...
+
+.. code-block:: sh
+
+   --config-path string           path to the configuration directory. If "$HOME/.mmctl" exists it will take precedence over the default value (default "$XDG_CONFIG_HOME")
+   --format string                the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate   allows to use insecure TLS protocols, such as SHA-1
+   --local                        allows communicating with the server through a unix socket
+   --strict                       will only run commands if the mmctl version matches the server one
+
+
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:4170]:
```suggestion
```

File: source/administration/mmctl-cli-tool.rst [hunk 19]
```diff
@@ -4004,6 +4088,321 @@ mmctl version
...
```
Comment (justinegeffen) [inline source/administration/mmctl-cli-tool.rst:4327]:
```suggestion
```

File: source/cloud/cloud-administration/sso-ldap.md [hunk 20]
```diff
@@ -22,57 +22,51 @@ Benefits of integrating AD/LDAP with Mattermost include:
...
+**Note:** The AD/LDAP sync depends on email. Make sure all users on your AD/LDAP server have an email address or that their account is deactivated in Mattermost.
 
 To configure AD/LDAP synchronization with AD/LDAP sign-in:
 
 1. Go to **System Console > Authentication > AD/LDAP** and set **Enable Synchronization with AD/LDAP** to **true**.
 
 2. Scroll down to **Synchronization Interval (minutes)** to specify how often Mattermost accounts synchronize attributes with AD/LDAP. The default setting is 60 minutes. The profile picture attribute is only synchronized when the user logs in.
-
-If you want to synchronize immediately after disabling an account, use the **AD/LDAP Synchronize Now** button in **System Console > AD/LDAP**.
+     - If you want to synchronize immediately after disabling an account, use the **AD/LDAP Synchronize Now** button in **System Console > AD/LDAP**.
```
Comment (justinegeffen) [inline source/cloud/cloud-administration/sso-ldap.md:65]:
```suggestion
     - If you want to synchronize immediately after disabling an account, select **AD/LDAP Synchronize Now** in **System Console > AD/LDAP**.
```

File: source/deployment/sso-saml-ldapsync.rst [hunk 21]
```diff
@@ -10,12 +10,13 @@ To configure SAML synchronization with AD/LDAP:
...
 
- - If you don't want to enable AD/LDAP sign-in, keep **Enable sign-in with AD/LDAP** as ``false``.
+ - If you don't want to enable AD/LDAP sign-in, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and keep **Enable sign-in with AD/LDAP** as ``false``.
 
-4. Set **Syncronization Interval** to specify how often Mattermost synchronizes SAML user accounts with AD/LDAP. The default setting is 60 minutes. If you want to synchronize immediately after disabling an account, use the "AD/LDAP Synchronize Now" button in **System Console > AD/LDAP** in prior versions or **System Console** > **Authentication** > **AD/LDAP** in versions after 5.12.
-5. To test Mattermost can successfully connect to your AD/LDAP server, click the **AD/LDAP Test** button.
+4. To specify how often Mattermost synchronizes SAML user accounts with AD/LDAP, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and set **Synchronization Interval**. The default setting is 60 minutes. If you want to synchronize immediately after disabling an account, click **AD/LDAP Synchronize Now**.
```
Comment (justinegeffen) [inline source/deployment/sso-saml-ldapsync.rst:18]:
```suggestion
4. To specify how often Mattermost synchronizes SAML user accounts with AD/LDAP, go to **System Console > Authentication > AD/LDAP** (or **System Console > AD/LDAP** in versions prior to 5.12) and set **Synchronization Interval**. The default setting is 60 minutes. If you want to synchronize immediately after disabling an account, select **AD/LDAP Synchronize Now**.
```

**Pull request merged**